public/js/admin.js

```diff
@@ -182,8 +182,9 @@ var addMCAnswers = function(dom){
 // replace the answer field in Question-creation.pug for specific question
 var getQuestionFormAnswer = function(form){
     $.ajax({
-        type: 'GET',
-        url: form,
+        type: 'POST',
```

Comment:
any reason for POST. You can only getting the information, you are not modifying any data in the backend?

Reply:
This is because I need to send some information to the back end, if you see below in data I am sending the type of the form I want

Reply:
cool, you can do that with the GET request as well